### PR TITLE
23845 - Bumped version numbers up for release 23.2a

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#23845

*Description of changes:*

- Legal API v2.130.0
- Entity-bn v2.129.0
- Furnishing jobs ( no version )
- Dissolution jobs ( no version )
- Business-pay ( no version )

[Release Report 
](https://app.zenhub.com/workspaces/entities---david-65af11a89e89f5043c2911f6/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvMTA1Mzgw)

[Github release](https://github.com/bcgov/lear/releases/edit/untagged-f33b7bef3d18e5741e2e)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by chenhongjing** [comment]
Missing entity-bn?

**Comment by eason-pan-bc** [comment]
I think, you may missed the version number change in `lear/queue_services/entity-bn/src/entity_bn/version.py` 
it's been changed in https://github.com/bcgov/lear/pull/3026

File: queue_services/entity-bn/src/entity_bn/version.py [hunk 1]
```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.128.0'  # pylint: disable=invalid-name
+__version__ = '2.129.0'  # pylint: disable=invalid-name
```
Comment (vysakh-menon-aot) [inline queue_services/entity-bn/src/entity_bn/version.py:25]:
```suggestion
__version__ = '2.130.0'  # pylint: disable=invalid-name
```

Comment (kzdev420) [reply queue_services/entity-bn/src/entity_bn/version.py]: updated

**Review by JazzarKarim — APPROVED** [review]

**Review by eason-pan-bc — APPROVED** [review]

**Pull request merged**